src/providers/email.js

```diff
@@ -22,7 +23,20 @@ export default (options) => {
   }
 }
 
-const sendVerificationRequest = ({ identifier: email, url, baseUrl, provider }) => {
+export const defaultTranslations = {
+  subject: 'Sign in to %s',
+  text: 'Sign in to %s',
+  html: {
+    signInAs: 'Sign in as <strong>%s</strong>',
+    signIn: 'Sign In',
+    didNotRequestHint: 'If you did not request this email you can safely ignore it.'
+  }
+}
+
+const sendVerificationRequest = ({ identifier: email, url, baseUrl, provider, locale, translations }) => {
+  // merge default and client provided translations
+  const texts = merge({}, defaultTranslations, translations?.[locale]?.email)
```

Comment:
As long as translations are flat, I think we get away really nicely with only using the spread `...` operator. If we have to deep merge objects, I would still prefer if we wrote a simple utility in another file rather than adding a new dependency.
```suggestion
  const texts = {...defaultTranslations, ...translations?.[locale]?.email }
```

Reply:
I think it's better to keep the nested/scoped translation structure, as its clearer to the end user as to what translations are used where. I'll rework this to avoid the extra dependency.